unitTests/ui/ComponentEventsTest.ts

```diff
@@ -64,6 +64,33 @@ export function ComponentEventsTest() {
         expect(spy).toHaveBeenCalledTimes(1);
         expect(spy).toHaveBeenCalledWith({ bar: 'baz' });
       });
+
+      describe('when useNativeJavaScriptEvents is set to true', () => {
+        beforeEach(() => {
+          Dom.useNativeJavaScriptEvents = true;
+        });
+
+        afterEach(() => {
+          Dom.useNativeJavaScriptEvents = null;
+        });
+
+        it('handle native events', () => {
+          test.env.root.click();
+
+          expect(spy).toHaveBeenCalledTimes(1);
+        });
+
+        it('handle jQuery events', () => {
+          $(test.env.root).click();
+
+          expect(spy).toHaveBeenCalled();
+        });
+
+        it('allow to trigger event', () => {
+          test.cmp.bind.trigger(test.env.element, 'click');
+          expect(spy).toHaveBeenCalled();
```

Comment:
For lines 86 and 91, why not check that the spy is called exactly one time like on line 80?